webapp/src/containers/Rhs/projectList/index.tsx

```diff
@@ -62,11 +65,12 @@ const ProjectList = () => {
     });
 
     const {data, isSuccess, isLoading} = getApiState(plugin_constants.pluginApiServiceConfigs.getAllLinkedProjectsList.apiServiceName);
-    const projectsList = data as ProjectDetails[];
+    const projectsList = data as ProjectDetails[] ?? [];
+    const sortedProjectList = useMemo(() => [...projectsList].sort(sortProjectList), [projectsList]);
```

Comment:
This will still be recreated on each render since the dependency will change again and again. And hence will force the useEffect(whose dependency is `sortedProjectList`) to run on each render.

We can add a to-do comment to memoize usePluginAPI's methods and then memoize these kinds of `object type` values(which we use in `useEffect's dependency`) or we use `primitive` values in the dependency.